scripts/variantstore/wdl/extract/run_gvs_tieout_extract.sh

```diff
@@ -1,7 +1,7 @@
 PROJECT="spec-ops-aou"
 DATASET="gvs_tieout_acmg_v1"
 
-python ngs_cohort_extract.py \
+python create_cohort_data_table.py \
```

Comment:
suggestion to name this `create_cohort_extract_data_table.py` to be super clear

Reply:
👍